zebedee-cms/src/main/java/com/github/onsdigital/zebedee/service/ImageServicePublishingResult.java

```diff
@@ -0,0 +1,44 @@
+package com.github.onsdigital.zebedee.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ImageServicePublishingResult {
+    private int totalImages;
+    private List<ImageDetails> unpublishedImages = new ArrayList<>();
+
+    public ImageServicePublishingResult(int totalImages) {
```

Comment:
Do we care if this is a negative number? What would happen to the code using this data if it returned a negative value? Possibly an unlikely scenario but I thought I'd ask the question all the same.